__tests__/AccountSettings.js

```diff
@@ -0,0 +1,77 @@
+import {
+  AWSInfoResponse,
+  getPersistedMockCall,
+  userResponse,
+} from 'testUtils/mockHttpCalls';
+import { renderRouteWithStore } from 'testUtils/renderUtils';
+
+const elementIDs = {
+  ChangeEmailForm: 'account-settings/change-email',
+  ChangePasswordForm: 'account-settings/change-password',
+  DeleteAccount: 'account-settings/delete-account',
+
+  EmailLabel: 'account-settings/email-label',
+  EmailDescription: 'account-settings/email-description',
+};
+
+const requests = {};
+
+describe('AccountSettings', () => {
+  beforeAll(() => {
+    requests.user = getPersistedMockCall('/v4/user/', userResponse);
+    requests.info = getPersistedMockCall('/v4/info/', AWSInfoResponse);
+    requests.orgs = getPersistedMockCall('/v4/organizations/');
+    requests.clusters = getPersistedMockCall('/v4/clusters/');
+    requests.catalogs = getPersistedMockCall('/v4/appcatalogs/');
+  });
+
+  afterAll(() => {
+    for (const req of Object.values(requests)) {
+      req.persist(false);
+    }
+  });
+
+  it('renders the account settings component on the "/account-settings" route', async () => {
+    const { findByText } = renderRouteWithStore('/account-settings', {});
+
+    await findByText('Your Account Settings');
+  });
+
+  it('renders the email change form', async () => {
+    const { findByTestId } = renderRouteWithStore('/account-settings', {});
+
+    const emailChangeForm = await findByTestId(elementIDs.ChangeEmailForm);
```

Comment:
I think `data-test-id` should be our last option. In case of form fields it makes more sense to use `findByDisplayValue` IMO

Reply:
This actually checks if the wrapper exists, actual form tests are separate for both `ChangeEmailForm` and `ChangePasswordForm`

Reply:
Yes, I picked a bad example here, I am ok with using this for wrappers.